frontend/public/components/_detail-table.scss

```diff
@@ -57,3 +57,7 @@
     text-align: left;
   }
 }
+
+.image-stream-manipulation__tooltip {
```

Comment:
We should update the classname to match the component name. Please use the `co-` prefix.

This seems like the wrong file for this :/